var/spack/repos/builtin/packages/vasp/package.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from llnl.util import tty
+import os
+
+
+class Vasp(MakefilePackage):
+    """
+    The Vienna Ab initio Simulation Package (VASP)
+    is a computer program for atomic scale materials modelling,
+    e.g. electronic structure calculations
+    and quantum-mechanical molecular dynamics, from first principles.
+    """
+
+    homepage = "http://vasp.at"
+    url      = "file://{0}/vasp.5.4.4.tar.gz".format(os.getcwd())
+
+    version('5.4.4', sha256='5bd2449462386f01e575f9adf629c08cb03a13142806ffb6a71309ca4431cfb3') # noqa
```

Comment:
```suggestion
    version('5.4.4', sha256='5bd2449462386f01e575f9adf629c08cb03a13142806ffb6a71309ca4431cfb3')
```
We exclude checksums from line length checks.